software/spooty.yml

```diff
@@ -0,0 +1,12 @@
+name: Spooty
+website_url: https://github.com/Raiper34/spooty
+description: Download tracks/playlists/albums from the Spotify url. It can also subscribe to a playlist or author page and download new songs upon release.
```

Comment:
```suggestion
description: Download tracks/playlists/albums from Spotify. It can also subscribe to a playlist or author page and download new songs upon release.
```